docs/build/build-transaction-construction.md

```diff
@@ -35,6 +35,27 @@ function from the Balances pallet will take:
 - `dest`: Destination address
 - `#[compact] value`: Number of tokens (compact encoding)
 
+**Serialized transaction format**
+
+Before being submitted, transactions are serialized. Serialized transactions are hex encoded SCALE-encoded bytes with the following format:
+- Compact encoded number of SCALE-encoded bytes following this.
+- 1 bit: it is a 0 if no signature is present, or a 1 if it is.
+- 7 bits: the transaction protocol version.
+- If there is a signature:
+  - a SCALE encoded `sp_runtime::MultiAddress::Id<AccountId32, u32>` indicating the signer(s) of the transaction.
+  - a SCALE encoded `sp_runtime::MultiSignature::S225519` with the signature.
```

Comment:
Polkadot supports sr25519, ed25519, and ecdsa. This is just an example using sr25519 (with a typo :)), so the text should either indicate that or just make the example generic.
```suggestion
  - a SCALE encoded `sp_runtime::MultiSignature::{SigningScheme}` with the signature.
```

Reply:
good point, updated and added a little clarification.